osm2gtfs/tests/README.md

```diff
@@ -0,0 +1,23 @@
+osm2gtfs tests
+==============
+
+Accra
+------
+#### Description
+
+There are 3 unittests for the Accra GTFS generation
+
+1. Generation of the cache stops Data from an Overpass result Mock
+* Generation of the cache routes Data from an Overpass result Mock
+* Generation of the GTFS ZIP File using the previously generated cache data
```

Comment:
The enumeration does seem to be broken here.

Reply:
sorry for that, my markdown editor was ok with this syntaxe